Description: hyphen-used-as-minus-sign
 This manual page seems to contain a hyphen where a minus sign was 
 intended.
Author: Ruben Molina <rmolina@udea.edu.co>
Forwarded: no
Last-Update: 2011-06-25

Index: kst-2.0.3/src/kst/kst.1.txt
===================================================================
--- kst-2.0.3.orig/src/kst/kst.1.txt
+++ kst-2.0.3/src/kst/kst.1.txt
@@ -47,7 +47,7 @@ kst may be run entirely from within its
 command line arguments are needed.  kst has a powerful wizard for
 easily setting up new plots.  
 
-In the second invocation, kst loads pre-saved plot specifications from the
+In the second invocation, kst loads pre\-saved plot specifications from the
 file
 .I kstfile
 and optionally may override some of the settings in that file.
@@ -64,7 +64,7 @@ plot all curves in separate plots
 .TP
 .B \-a\fR
 average frames (simple mean) when plotting with
-.B -s\fR.
+.B \-s\fR.
 .TP
 .B \-\-A4\fR
 use A4 sized paper for printing.  Requires
@@ -90,7 +90,7 @@ and instead use
 .B \-f\ STARTFRAME\fR
 begin plotting at
 .I STARTFRAME
-frames into the data.  Set to -1 to start at
+frames into the data.  Set to \-1 to start at
 .I NUMFRAMES
 from the end of the data.
 .TP
@@ -112,7 +112,7 @@ use US Letter sized paper for printing.
 .B \-n\ NUMFRAMES\fR
 plot at most
 .I NUMFRAMES
-frames of data.  Set to -1 to indicate all of the data.
+frames of data.  Set to \-1 to indicate all of the data.
 .TP
 .B \-P\ PLOTNAME\fR
 plot all plots in the plot named
@@ -145,7 +145,7 @@ frames when plotting.
 .B \-x\ FIELD\fR
 use
 .I FIELD
-as the X-axis vector.
+as the X\-axis vector.
 .TP
 .B \-y\ FIELD\fR
 plot
@@ -158,30 +158,30 @@ plot
 as an image.
 .SH EXAMPLES
 Plot all data in column 2 from data.dat.
-       kst data.dat -y 2
+       kst data.dat \-y 2
 
 Same as above, except only read 20 lines, starting at line 10.
-       kst data.dat -f 10 -n 20 -y 2
+       kst data.dat \-f 10 \-n 20 \-y 2
 
 also read col 1. One plot per curve.
-       kst data.dat -f 10 -n 20 -y 1 -y 2
+       kst data.dat \-f 10 \-n 20 \-y 1 \-y 2
 
 Read col 1 from data2.dat and col 1 from data.dat
-       kst data.dat -f 10 -n 20 -y 2 data2.dat -y 1
+       kst data.dat \-f 10 \-n 20 \-y 2 data2.dat \-y 1
 
 Same as above, except read 40 lines starting at 30 in data2.dat
-       kst data.dat -f 10 -n 20 -y 2 data2.dat -f 30 -n 40 -y 1
+       kst data.dat \-f 10 \-n 20 \-y 2 data2.dat \-f 30 \-n 40 \-y 1
 
 Specify the X vector and error bars:
 Plot x = col 1 and Y = col 2 and error flags = col 3 from data.dat
-       kst data.dat -x 1 -e 3 -y 2
+       kst data.dat \-x 1 \-e 3 \-y 2
 
 Get the X vector from data1.dat, and the Y vector from data2.dat.
-       kst data1.dat -x 1 data2.dat -y 1
+       kst data1.dat \-x 1 data2.dat \-y 1
 
 Placement:
 Plot column 2 and column 3 in plot P1 and column 4 in plot P2
-       kst data.dat -P P1 -y 2 -y 3 -P P2 -y 4
+       kst data.dat \-P P1 \-y 2 \-y 3 \-P P2 \-y 4
 .SH BUGS
 Please report bugs to either the kst mailing list at
 .I kst@kde.org
@@ -198,6 +198,6 @@ Matthew Truch <matt@truch.net>
 Permission is granted to copy, distribute and/or modify this document
 under the terms of the GNU Free Documentation License, Version 1.2 or
 any later version published by the Free Software Foundation; with no
-Invariant Sections, with no Front-Cover Texts, and with no Back-Cover
+Invariant Sections, with no Front\-Cover Texts, and with no Back\-Cover
 Texts.  A copy of the license is included in the `COPYING.DOC' file
 as part of the kst distribution.
